jenkins/manifests/jenkins-rc.yaml

```diff
@@ -0,0 +1,57 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/jenkins-rc.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: jenkins
+  labels:
+    app: jenkins
+    provider: jenkins-server
+    heritage: bitnami
+spec:
+  replicas: 1
+  selector:
+    app: jenkins
+    provider: jenkins-server
+    version: 2.17-r0
+  template:
+    metadata:
+      labels:
+        app: jenkins
+        provider: jenkins-server
+        version: 2.17-r0
+        heritage: bitnami
+    spec:
+      containers:
+      - name: jenkins
+        image: bitnami/jenkins:2.17-r0
+        env:
+        - name: JENKINS_USERNAME
+          value: "user"
+        - name: JENKINS_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: jenkins
+              key: jenkins-password
+        ports:
+        - name: http
+          containerPort: 8080
+        - name: https
+          containerPort: 8443
+        livenessProbe:
+          httpGet:
+            path: /
+            port: http
+          initialDelaySeconds: 120
+          timeoutSeconds: 5
+        readinessProbe:
+          httpGet:
+            path: /
+            port: http
+          initialDelaySeconds: 5
+          timeoutSeconds: 1
+        volumeMounts:
+        - name: jenkins-data
```

Comment:
@jotadrilo we generally use the name `data` in all the charts. As a matter of convention it would nice if this and all its references were updated to `data`